RFS/src/main/java/com/rfs/common/DocumentReindexer.java

```diff
@@ -50,9 +50,9 @@ private String convertToBulkRequestBody(List<String> bulkSections) {
         return builder.toString();
     }
 
-    public void refreshAllDocuments(ConnectionDetails targetConnection) throws Exception {
+    public void refreshAllDocuments(ConnectionDetails targetConnection) {
         // Send the request
-        OpenSearchClient client = new OpenSearchClient(targetConnection);
-        client.refresh();
+        OpenSearchClient refreshClient = new OpenSearchClient(targetConnection);
+        refreshClient.refresh();
```

Comment:
Was there a rule about the naming? Client seems like a perfectly cromulent name

Reply:
It was overlapping with the class member of the same name.  It feels like this method probably shouldn't exist as it does, but I decided to leave it so that I can focus on the SonarQube findings more specifically.